spec/controllers/talents_controller_spec.rb

```diff
@@ -0,0 +1,159 @@
+require 'rails_helper'
+
+RSpec.describe TalentsController, type: :controller do
+  describe "GET #index" do
+    before(:each) do
+      get :index
+    end
+
+    it "renders the correct template" do
+      expect(@response).to render_template :index
+    end
+
+    it { should respond_with 200 }
+  end
+
+  describe "GET #show" do
+    before(:each) do
+      @talent = create :talent_with_sources
+      get :show, params: { id: @talent.id }
+    end
+
+    it "assigns the requested talent to @talent" do
+      expect(assigns(:talent)).to eq(@talent)
+    end
+
+    it "renders the correct template" do
+      expect(@response).to render_template :show
+    end
+
+    it { should respond_with 200 }
+  end
+
+  describe "GET #new" do
+    before(:each) do
+      get :new
+    end
+
+    it "assigns a new talent to @talent" do
+      expect(assigns(:talent)).not_to eql nil
+    end
+
+    it "renders the correct template" do
+      expect(@response).to render_template :new
+    end
+
+    it { should respond_with 200 }
+  end
+
+  describe "GET #edit" do
+    before(:each) do
+      @talent = create :talent_with_sources
+      get :edit, params: { id: @talent.id } 
+    end
+
+    it "assigns the requested talent to @talent" do
+      expect(assigns(:talent)).to eq(@talent)
+    end
+
+    it "renders the correct template" do
+      expect(@response).to render_template :edit
+    end
+
+    it { should respond_with 200 }
+  end
+
+  describe "POST #create" do
+    context "with valid attributes" do
+
+      before(:each) do
+        @talent_attrs = FactoryGirl.attributes_for(:talent_with_sources)
```

Comment:
Since we have configured `FactoryGirl` to use a [short syntax](https://github.com/Internship2017/star_wars_ffg/blob/master/spec/rails_helper.rb#L36:L37), we can narrow this to:

```ruby
@talent_attrs = attributes_for(:talent_with_sources)
```